walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -1,25 +1,115 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
+@media screen and (min-width: 600px)
+{
+  .header-container
+  {
```

Comment:
Brackets should go after the selector like:

```css
.header-container {
  display: flex;
  ...
}
```

here and below.

Reply:
Prettier should solve all these formatting issues.

Reply:
done